src/plugins/intel_cpu/src/nodes/executors/acl/acl_mvn.cpp

```diff
@@ -44,52 +44,32 @@ bool ACLMVNExecutor::supports(const MVNConfig& config) {
               " epsMode=",
               static_cast<int>(config.attrs.epsMode_));
 
-    const bool unsupported_src_precision = srcPrecision != ov::element::f32 && srcPrecision != ov::element::f16;
-    const bool unsupported_dst_precision = dstPrecision != ov::element::f32 && dstPrecision != ov::element::f16;
-
-    if (unsupported_src_precision || unsupported_dst_precision) {
-        DEBUG_LOG("ACL MVN: Unsupported precision");
-        return false;
-    }
+    VERIFY((srcPrecision == ov::element::f32 || srcPrecision == ov::element::f16) &&
+               (dstPrecision == ov::element::f32 || dstPrecision == ov::element::f16),
+           UNSUPPORTED_SRC_PRECISIONS);
 
     // Input and output precisions must match
-    if (srcPrecision != dstPrecision) {
-        DEBUG_LOG("ACL MVN: Precision mismatch");
-        return false;
-    }
+    VERIFY(srcPrecision == dstPrecision, UNSUPPORTED_DST_PRECISIONS);
 
-    if (config.attrs.epsMode_ == MVNEpsMode::OUTSIDE_SQRT) {
-        DEBUG_LOG("ACL MVN: OUTSIDE_SQRT not supported");
-        return false;
-    }
-    if (!config.attrs.normalizeVariance_) {
-        DEBUG_LOG("ACL MVN: normalize_variance=false not supported");
-        return false;
-    }
+    // ACL supports only INSIDE_SQRT with normalizeVariance=true
+    VERIFY(config.attrs.epsMode_ != MVNEpsMode::OUTSIDE_SQRT, UNSUPPORTED_BY_EXECUTOR);
+    VERIFY(config.attrs.normalizeVariance_, UNSUPPORTED_BY_EXECUTOR);
 
     // Check layout compatibility
     const bool ncsp_mismatch = srcDesc->hasLayoutType(LayoutType::ncsp) && !dstDesc->hasLayoutType(LayoutType::ncsp);
```

Comment:
can we reverse the logic?
Like to check that both src and dst layouts are equal